components/post_view/post_view.jsx

```diff
@@ -67,7 +67,10 @@ export default class PostView extends React.PureComponent {
         }
 
         return (
-            <div id='post-list'>
+            <div
+                id='post-list'
+                role='form'
```

Comment:
Is this the one for "The post component (on both center and RHS) could be role='form'"? I thought it's the textbox on center and RHS but others can confirm.

Reply:
Yes, that's correct. This should be `role='main'`